includes/admin/class-wc-rest-stripe-settings-controller.php

```diff
@@ -276,11 +276,13 @@ public function get_settings() {
 	 * @param WP_REST_Request $request Full data about the request.
 	 */
 	public function update_settings( WP_REST_Request $request ) {
+		$form_fields = $this->gateway->get_form_fields();
```

Comment:
I started off by placing `$this->gateway->get_form_fields()` in the `validate_field()` function but ultimately decided against it because I figured it would add a bit of overhead. Namely, each time `$this->gateway->get_form_fields()` is called, the following happens:
* Maps each element in `$form_fields` to ensure the `default` key is set.
* Calls `apply_filters( 'woocommerce_settings_api_form_fields_stripe', $form_fields )`.

If we did this for each form field, it would result in an `O(n^2)` runtime. So, I figured if we called it once, from `update_settings( ... )`, it would avoid this overhead (`O(n)` runtime). But since we do not need to call `get_form_fields()` for each form field, it should be ok to place it in `validate_field()` and tradeoff a bit of performance for code readability.